packages/@react-spectrum/checkbox/src/CheckboxGroup.tsx

```diff
@@ -92,6 +96,15 @@ function CheckboxGroup(props: SpectrumCheckboxGroupProps, ref: DOMRef<HTMLDivEle
           </CheckboxGroupContext.Provider>
         </Provider>
       </div>
+      {hasHelpText &&
+        <HelpText
+          descriptionProps={descriptionProps}
+          errorMessageProps={errorMessageProps}
+          description={description}
+          errorMessage={errorMessage}
+          validationState={validationState}
+          showErrorIcon={showErrorIcon} />
```

Comment:
Mind adding a story or chromatic story for `showErrorIcon`?

Reply:
added